third_party/gpus/rocm_configure.bzl

```diff
@@ -173,6 +173,10 @@ def _rocm_include_path(repository_ctx, rocm_config):
     inc_dirs.append("/opt/rocm/include/hip/hcc_detail")
     inc_dirs.append("/opt/rocm/hip/include")
 
+    # Add HIP-Clang headers
+    inc_dirs.append("/opt/rocm/llvm/lib/clang/8.0/include")
```

Comment:
8.0.0?

Reply:
This is for hip-clang built with HIP/VDI runtime. Its version is 8.0 for some reason.